bigtable-hbase-parent/bigtable-hbase-integration-tests/pom.xml

```diff
@@ -232,6 +232,16 @@ limitations under the License.
 
     <dependencies>
         <dependency>
+            <groupId>${project.groupId}</groupId>
```

Comment:
These shouldn't be here.  it would be ok to use the bigtableIntegrationTest profile for running ManyThreadDriver.  This would have the impact of having the bigtable dependencies when we run the integration test with -P hbaseLocalMiniClusterTest which may cause problems running the HBase tests.